app/src/org/commcare/google/services/analytics/CCAnalyticsParamValue.java

```diff
@@ -0,0 +1,118 @@
+package org.commcare.google.services.analytics;
+
+/**
+ * Created by amstone326 on 10/13/17.
+ */
+
+public class CCAnalyticsParamValue {
```

Comment:
What do you think about renaming to just `AnalyticsParamValue`? We used to have "CommCare" before every class but then someone (I think/assume Phillip) removed them because they're a bit redundant (IE we're already in `CommCare`)

Reply:
Ah yea so for this class specifically, I think that makes sense. For `CCAnalyticsEvent` and `CCAnalyticsParam`, I had added the `CC` prefix specifically to make the differentiation between default firebase values and custom CommCare values clearer. Since for events and parameters, Firebase provides some default options (e.g. `FirebaseAnalytics.Param.VALUE`), but I'm also using a lot of custom ones (e.g. `CCAnalyticsParam.ACTION`). I'm not sure if you meant this comment to apply to all 3 classes, or just this one, so I'll go ahead and change this one now, and let me know what you think about my reasoning for the other 2.